docker-build/README.md

```diff
@@ -0,0 +1,152 @@
+# GitHub Action Release
+
+<!-- prettier-ignore-start -->
+<!-- action-docs-description -->
+## Description
+
+Only builds docker images for the input platform, tags and image version
+<!-- action-docs-description -->
+<!-- prettier-ignore-end -->
+
+## Usage
+
+This action will build a docker image with appropriate tags.
+
+It assumes that the repo has a configuration file in `<repo-root>/.docker-config.json` with the following properties:
+
+- `imageName`: the image name (org/image-name)
+- `dockerfile`: the path to the dockerfile to build - defaults to `./Dockerfile`
+
+Example:
+
+```json
+{
+  "imageName": "turo/hello-world-msvc",
+  "dockerfile": "./Dockerfile"
+}
+```
+
+#### Basic Usage:
+
+```yaml
+steps:
+  - uses: open-turo/actions-jvm/release@v3
+    name: Release
+    id: release
+    with:
+      checkout-repo: true
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+      dry-run: false
+  - uses: open-turo/actions-jvm/build-docker@v1
```

Comment:
This docs seem to not be in sync with this action

Reply:
I would update the readme so these docs are generated using action docs. See https://github.com/open-turo/actions-go/blob/main/script/update-action-readme and some of the readmes in that repo